src/www/ui/core-auth.php

```diff
@@ -252,6 +260,31 @@ function OutputOpen()
    */
   function checkUsernameAndPassword($userName, $password)
   {
+    /* Check the user for external authentication */
+    if ($this->authExternal['useAuthExternal']) {
+      $username = $this->authExternal['loginAuthExternal'];
+      /* checking if user exist */
+      try {
+        $this->userDao->getUserAndDefaultGroupByUserName($username);
+      } catch (Exception $e) {
+        /* If user does not exist then we create it */
```

Comment:
Can this part be configurable with `fossology.conf` or some other way to restrict user creation?